### PR TITLE
Cleanup dependencies in changelog

### PR DESCRIPTION
**Pull request by chymy**
Signed-off-by: chymy <chang.min1@zte.com.cn>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Contains duplicate `Dependencies` , words and parameters misspelling in changelog-1.20 and  changelog-1.19.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by chymy** [comment]
Please take a look, Thanks.
/assign @hasheddan @feiskyer

File: CHANGELOG/CHANGELOG-1.20.md [hunk 1]
```diff
@@ -70,10 +70,6 @@
     - [Added](#added-2)
     - [Changed](#changed-2)
     - [Removed](#removed-2)
-  - [Dependencies](#dependencies-3)
```
Comment (hasheddan) [inline CHANGELOG/CHANGELOG-1.20.md:73]: I think removing the dependencies section below is going to break the dependency links in the subsequent sections, so they likely need to be updated (i.e. #dependencies-4 would now be #dependencies-3)

**Review by hasheddan — CHANGES_REQUESTED** [review]

File: CHANGELOG/CHANGELOG-1.20.md (hunk 1 above)
Comment (chymy) [reply CHANGELOG/CHANGELOG-1.20.md:73]: Thank you very much @hasheddan, fixed.

**Comment by chymy** [comment]
Can you take a look?
Thanks.
/assign @wilsonehusin

**Comment by wilsonehusin** [comment]
@chymy, please incorporate #98652 and #98654 in this PR together.

**Comment by chymy** [comment]
> @chymy, please incorporate #98652 and #98654 in this PR together.

Thank you very much @wilsonehusin, fixed.

**Pull request merged**